README.md

```diff
@@ -1,2 +1,77 @@
-# connectiq-apps
-A collection of Connect IQ apps and libraries.
+# /connectiq-apps
+_A collection of Connect IQ apps and libraries_
+
+------------------
+
+## Purpose
+This repo is officially maintained by the Garmin&reg; Connect IQ team. We are happy to have you here. It is our hope that this repository will extend the  resources available to our developers to help them build exciting new content!
+
+----------------------
+
+## What's in the Repo?
+
+This repo contains several resources for developers to help in the development process including examples and Garmin Monkey Barrel Code libraries:
```

Comment:
Should we say a little more along the lines of this being examples for reference, and not something that will be regularly augmented? I'm not sure... I just don't want developers to get the idea that we're going to be frequently growing a collection of example apps here.

Reply:
Reworded just a little to denote a small set for reference.